### DISCUSSION TITLE
Is it time for phase III?

### DISCUSSION THREAD

**Discussion by fersebas** [PaddlePaddle/PaddleSpatial#280] [BaiduKDDCup2022] [answered]
![image](https://user-images.githubusercontent.com/49498032/179343346-23c65d4f-ea0f-40bb-9a0f-ec1d7d685482.png)

But AOE Time is still 15 july ... 
![image](https://user-images.githubusercontent.com/49498032/179343359-b5c79c83-2e9c-4579-a011-33707f005614.png)

So it should still let you upload a file to phase 2....

Can we now submit to phase 3 or what?

**Comment by ramber1836** [comment, marked as answer]
We sent all the competitors a notification email earlier, which says as below: 

"Phase 2 test will come to an end on July 15, 2022, at 15:59:59 (AOE time).  From July 16th 00:01 AM to July 18th 11: 59 PM (AOE time), we will go to Phase 3 of the Baidu KDD Cup. ... "

So between 16:00 July 15, 2022 (AoE Time), and 00:00 July 16, 2022 (AoE Time), there is a break during which we can allocate the new computational resources and test the online evaluation environment accordingly.

**Reply by fersebas** [reply]
Ok sorry I didn't receive any email. Understand!

**Reply by ramber1836** [reply]
No problem. 
I checked our records, maybe you recorded your email with some typos? 
Is your email 'ferseb39@gmaill.com'?

**Reply by fersebas** [reply]
No, it is ferseb39@gmail.com. I think there is a extral “l” :s my fault
